src/Stratis.Bitcoin/Base/BackgroundWork/IAsyncDelegateWorker.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Stratis.Bitcoin.Base.BackgroundWork
+{
+    public interface IAsyncDelegateWorker : IDisposable { }
```

Comment:
This is not so useful is it? AsyncQueue is already IDisposable

Reply:
in my implementation I'm not returning AsyncQueue but the interface, so the interface need to implement IDisposable (AsyncQueue is going to be splitted in 2 classes, so I've already done the job here)